main/src/ca/uwaterloo/flix/language/phase/Parser2.scala

```diff
@@ -604,7 +607,7 @@ object Parser2 {
     val itemCount = zeroOrMore(namedTokenSet, getItem, checkForItem, breakWhen, separation, delimiterL, delimiterR, optionallyWith)
     val locAfter = previousSourceLocation()
     if (itemCount < 1) {
-      val loc = SourceLocation(isReal = true, locBefore.sp1, locAfter.sp1)
+      val loc = SourceLocation(isReal = true, s.src, locBefore.sp1, locAfter.sp1)
```

Comment:
I understand that this is the same source that `currentSourceLocation` and `previosSourceLocation` get their source from, but it seems off to use `s` rather than using the source of `locBefore` or `locAfter`. But then again, if it's always within the same source then it doesn't really matter.